RELEASE_PROCESS.md

```diff
@@ -108,20 +81,20 @@ docker build docker --pull -t ponylang-playpen
 systemctl start playground
 ```
 
-### Inform the Pony Zulip
+## If something goes wrong
```

Comment:
Shouldn't this be an h3?

Reply:
I don't think so but I'm not sure on that.

I'm going to merge as is and we can discuss more later.